src/Uno.Toolkit.UI/Controls/ExtendedSplashScreen/ExtendedSplashScreen.cs

```diff
@@ -58,7 +58,7 @@ internal object SplashScreenContent
 	protected static ExtendedSplashScreen? Instance { get; private set; }
 
 	public
-#if __IOS__ || __MACOS__ // hides UIView.Window and NSView.Window
```

Comment:
what is the intention here?
winui should either on neither side or both sides, but not one sided